documentation/configuration/authentication.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: documentation
+menu: authentication
+subtitle: Authentication
+---
+
+# Authentication
+
+In order to log in to your database, the typical approach is to set your username and password in the Flyway [configuration file](/documentation/configuration/configfile). This however has some concerns:
```

Comment:
Looks good as a first pass, but perhaps worth running this by one of our advocates for best practice phrasing.